gong/read_test.go

```diff
@@ -0,0 +1,222 @@
+package gong
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/test/utils/mockutils"
+
+	"github.com/go-test/deep"
+)
+
+func TestRead(t *testing.T) { //nolint:funlen,gocognit,cyclop
+	t.Parallel()
+
+	fakeServerResp := mockutils.DataFromFile(t, "read.json")
+	fakeServerResp2 := mockutils.DataFromFile(t, "read_cursor.json")
+
+	tests := []struct {
+		name             string
+		input            common.ReadParams
```

Comment:
I don't see any of these test cases defining `input` at all, so it's unclear what is being tested here, since you're just seeing if the fake servers return the results you've defined.

Reply:
fixed